### ISSUE TITLE
Failure: system didn't notify us about a reboot

### ISSUE THREAD

**Issue by mvollmer** [cockpit-project/cockpit#2915]
#2909 f22 failed like this:

http://files.cockpit-project.org/hubbot/34ebb7c32adf83914dd45699a5d9db17c262745d_f22_x86-64/hubbot.html

```
Read from socket failed: Connection reset by peer

======================================================================
ERROR: testBasic (__main__.TestShutdownRestart)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "./check-shutdown-restart", line 45, in testBasic
    m.wait_reboot()
  File "/home/hubbot/hubbot/34ebb7c32adf83914dd45699a5d9db17c262745d_f22_x86-64/test/testvm.py", line 857, in wait_reboot
    raise Failure("system didn't notify us about a reboot")
Failure: system didn't notify us about a reboot
```

**Comment by stefwalter** [comment]
Another case of this at: http://files.cockpit-project.org/hubbot/9ec2072c5d4970ac7f0f741c2a6f35a434d04df5_f22_x86-64/hubbot.html

**Comment by dperpeet** [comment]
I think we should increase the timeout for now.

**Comment by dperpeet** [comment]
I'm looking into other ways to detect the reboot in libvirt

**Comment by dperpeet** [comment]
libvirt gets the reboot event from the qemu monitor
if this doesn't work properly, either the system didn't reboot or it's a bug in libvirt/qemu

**Comment by mvollmer** [comment]
It has happened again here:

http://files.cockpit-project.org/hubbot/b7bc929b6862c78a39041c70ff1ad4e804fd4da8_r7_x86-64/hubbot.html
https://github.com/cockpit-project/cockpit/pull/2862

That test run included 1af51ec.

**Comment by stefwalter** [comment]
Another one: http://files.cockpit-project.org/hubbot/a857cb8d348ab8e45f99b8d506b1d2a83578911e_f22_x86-64/hubbot.html

**Comment by stefwalter** [comment]
Another one: http://files.cockpit-project.org/hubbot/3d409253f97a44ce3b2b0780afc20505346825b8_f22_x86-64/hubbot.html

**Comment by stefwalter** [comment]
Another one: http://files.cockpit-project.org/hubbot/02981879096a666f0d720d1cacd32866bf10e85e_f22_x86-64/hubbot.html

**Comment by stefwalter** [comment]
Another case: http://files.cockpit-project.org/hubbot/2acb79b49d97742fde196e4e20fa65f9b3948f0e_f22_x86-64/hubbot.html

**Comment by stefwalter** [comment]
Another case: http://files.cockpit-project.org/hubbot/5dce2842a5adb8ae4fee97eb4ef5f7e6b8bb7f08_f22_x86-64/hubbot.html

**Comment by stefwalter** [comment]
Another case: http://files.cockpit-project.org/hubbot/b70b3976bd7c465e11de9d43c6a6cdc10cb3e109_f22_x86-64/hubbot.html

**Comment by stefwalter** [comment]
Another case: http://files.cockpit-project.org/hubbot/ab2f4adcf19cc4825a2a2eb096538e1edf62ec29_f22_x86-64/hubbot.html

**Comment by stefwalter** [comment]
Another case: http://files.cockpit-project.org/hubbot/5ae02cbe25556ae324c2906167fed201e139998c_f22_x86-64/hubbot.html

http://files.cockpit-project.org/hubbot/5ae02cbe25556ae324c2906167fed201e139998c_r7_x86-64/hubbot.html

**Comment by mvollmer** [comment]
Again, with #2941 applied:

http://files.cockpit-project.org/hubbot/f7e21e6505f08e0fdc4462a6d19532c9bf126b25_f22_x86-64/hubbot.html

**Comment by dperpeet** [comment]
This seems to have been fixed in #2948
